analysers/analyser_merge_man_made_FR.py

```diff
@@ -0,0 +1,130 @@
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from .Analyser_Merge_Dynamic import Analyser_Merge_Dynamic, SubAnalyser_Merge_Dynamic
+from .Analyser_Merge import Source, GPKG, LoadGeomCentroid, Mapping, Select, Generate
+
+
+class Analyser_Merge_Man_Made_FR(Analyser_Merge_Dynamic):
+
+    def __init__(self, config, logger = None):
+        Analyser_Merge_Dynamic.__init__(self, config, logger)
+
+        maping = [dict(
+            # Antenne
+            #   other data source
+
+            # Autre construction élevée
+            # Mât d'éclairage
+            item = 8490,
+            id = 100,
+            level = 3,
+            title = T_('Lighting mast not integrated'),
+            select = {'nature_detaillee': 'Mât d\'éclairage'},
+            tags = {'man_made': 'mast', 'tower:type': 'lighting'},
+        ), dict(
+            # Calvaire
+            #   OSM tag ?
+
+            # Cheminée
+            item = 8490,
+            id = 3,
+            level = 2,
+            title = T_('Chimney not integrated'),
+            select = {'nature': 'Cheminée'},
+            tags = {'man_made': 'chimney'},
+        ), dict(
+            # Clocher
+            item = 8100,
+            id = 4,
+            title = T_('Bell tower not integrated'),
+            level = 1,
+            select = {'nature': 'Clocher'},
+            tags = {'man_made': 'tower', 'tower:type': 'bell_tower'},
+        ), dict(
+            # Croix
+            item = 8100,
+            id = 5,
+            level = 3,
+            title = T_('Cross not integrated'),
+            select = {'nature': 'Croix'},
+            tags = {'man_made': 'cross'},
+        ), dict(
+            # Eolienne
+            item = 8270,
+            id = 6,
+            level = 2,
+            title = T_('Wind turbine not integrated'),
+            select = {'nature': 'Eolienne'},
+            tags = {'power': 'generator', 'generator:source': 'wind'},
+            height = False, # IGN: max height, OSM: center of rotor
+        ), dict(
+            # Minaret
+            item = 8100,
+            id = 7,
+            level = 1,
+            title = T_('Minaret not integrated'),
+            select = {'nature': 'Minaret'},
+            tags = {'man_made': 'tower', 'tower:type': 'minaret'},
+            # Phare (300)
+            #  not interested
+
+            # Puits d'hydrocarbures
+            #  OSM tag ?
+
+            # Torchère (200)
+            #  OSM tag ?
```

Comment:
man_made=flare https://wiki.openstreetmap.org/wiki/FR:Tag:man_made%3Dflare

Reply:
Low volume. I prefer skip this one.

Reply:
OSM 3 / IGN 200

Reply:
Done, Thank you.